firestore/tests/unit/test_cross_language.py

```diff
@@ -59,11 +65,23 @@ def run_write_test(self, test_proto, desc):
             client, doc = self.setup(firestore_api, tp)
             data = convert_data(json.loads(tp.json_data))
             call = functools.partial(doc.create, data)
+        elif kind == "get":
+            tp = test_proto.get
+            client, doc = self.setup(firestore_api, tp)
+            call = functools.partial(doc.get, None, None)
+            try:
+                tp.is_error
+            except AttributeError:
+                return
         elif kind == "set":
             tp = test_proto.set
             client, doc = self.setup(firestore_api, tp)
             data = convert_data(json.loads(tp.json_data))
-            # TODO: call doc.set.
+            if tp.HasField("option"):
+                option = True
```

Comment:
Is option the SetOptions or just "merge" (in which case it should be `mergeOption`)?